src/main/webapp/coffeeshop.html

```diff
@@ -15,8 +15,10 @@
    <a href="index.html" class="button">Home</a>
    <a href="user.html" class="button">Profile</a>
    <a href="store.html" id="active" class="button">Explore</a>
-    <a href="search.html" class="button">Search for a drink!</a>  
+   <a href="order.html" class="button">Order</a> 
```

Comment:
Switch Order and Search around so search comes first, just so we're consistent with our layout all throughout the website

Reply:
Done.